presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -60,45 +63,56 @@
     @BeforeTestWithContext
     public void setup()
     {
-        tableName = "alice_owned_table";
         aliceExecutor = connectToPresto("alice@presto");
         bobExecutor = connectToPresto("bob@presto");
 
-        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
-        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", TABLE_NAME));
```

Comment:
I agree that we should modify the tests at some point to enable running the tests in parallel. However, I don't see how changing the tests only in this file will achieve that. There are numerous other tests where parallelism wasn't considered while writing them. So we will have to start a much larger effort for that. For grant/revoke tests, I can do this in a new PR.